docs/src/library/rule.md

```diff
@@ -48,13 +48,13 @@ A rule has a:
 }) }}
 
  {{ rule({
-    name: "getter-return",
+    name: "no-arrow-condition",
     removed: true,
-    description: 'Enforce `return` statements in getters.',
-    replacedBy: "other-rule-here",
```

Comment:
here `other-rule-here` creates multiple links for every word that takes us nowhere.

![Screenshot 2024-05-30 164641](https://github.com/eslint/eslint/assets/86398394/2f4307e7-e6c2-4c09-b9ea-0b6b6587870e)